api/managementpb/dbaas/kubernetes.proto

```diff
@@ -19,14 +19,53 @@ message KubeAuth {
   ];
 }
 
+// OperatorsStatus defines status of operators installed in Kubernetes cluster.
+enum OperatorsStatus {
+  // OPERATORS_STATUS_INVALID represents unknown state.
+  OPERATORS_STATUS_INVALID = 0;
+  // OPERATORS_STATUS_OK represents that operators are installed and have supported API version.
+  OPERATORS_STATUS_OK = 1;
+  // OPERATORS_STATUS_UNSUPPORTED represents that operators are installed, but doesn't have supported API version.
+  OPERATORS_STATUS_UNSUPPORTED = 2;
+  // OPERATORS_STATUS_NOT_INSTALLED represents that operators are not installed.
+  OPERATORS_STATUS_NOT_INSTALLED = 3;
```

Comment:
Can this status be "initializing" as well?

Reply:
Why? Not installed means that it's not installed, initializing means that it will be installed. But it's not a part of this task.

Reply:
Cool, just wandering. 👍